Task 2.sql

```diff
@@ -0,0 +1,55 @@
+CREATE OR ALTER FUNCTION [dbo].[F_WORKS_LIST] (
+)
+RETURNS @RESULT TABLE
+(
+ID_WORK INT,
+CREATE_Date DATETIME,
+MaterialNumber DECIMAL(8,2),
+IS_Complit BIT,
+FIO VARCHAR(255),
+D_DATE varchar(10),
+WorkItemsNotComplit int,
+WorkItemsComplit int,
+FULL_NAME VARCHAR(101),
+StatusId smallint,
+StatusName VARCHAR(255),
+Is_Print bit
+)
+AS
+-- СПИСОК РАБОТ
+begin
+insert into @result
+SELECT TOP 3000
+  Works.Id_Work,

```

Comment:
Msg 8120, Level 16, State 1, Line 2
Column 'Works.Id_Work' is invalid in the select list because it is not contained in either an aggregate function or the GROUP BY clause.